packages/edit-site/src/components/global-styles/screen-typography-element.js

```diff
@@ -21,6 +21,17 @@ const elements = {
 	heading: {
 		description: __( 'Manage the fonts and typography used on headings.' ),
 		title: __( 'Headings' ),
+		hint: (
+			<>
+				{ __(
+					'Block headings like Site Title and Post Title should be managed in the '
+				) }
+				<strong>{ __( 'Styles' ) }</strong>
+				{ __( ' > ' ) }
+				<strong>{ __( 'Blocks' ) }</strong>
+				{ __( ' menu.' ) }
+			</>
+		),
```

Comment:
This feels like it needs better interpolation so the translations can make sense. I also think it's worth being a bit more newbie-friendly and providing clearer guidance.
```suggestion
		// import { createInterpolateElement } from '@wordpress/element';
		hint: createInterpolateElement(
			__( 'Some special headings like your Site Title and Post Title are managed separately via dedicated settings under the <strong>Styles</strong> > <strong>Blocks</strong> menu.' ),
			{ strong: <strong /> }
		),
```

It would be ideal if we could actually link to the section(s) directly, using some output that looks like the following:
> Some special headings have dedicated styles available from the following sections. If you want all headings to have the same font, please set those styles to the Default style.
> * [Site Title](#dummy-link) - the title for your site
> * [Post Title](#dummy-link) - the headings used for Pages and Posts on your site

Reply:
Thanks for `createInterpolateElement`. I am going to make this a separate PR and will try and include links to jump to those settings directly.